gateway/1.4/modules/ROOT/pages/_partials/task-config-liveness-probe.adoc

```diff
@@ -0,0 +1,69 @@
+//tag::intro1[]
+
+Flex Gateway includes the following CLI command to test gateway health. You can run this command manually on any Flex Gateway:
+
+[source,ssh]
+----
+flexctl probe --check=liveness
+----
+
+This command returns either an exit code of `0` for a healthy Flex Gateway or `1` for an unhealthy Flex Gateway.
+
+//end::intro1[]
+
+//tag::intro2[]
+You can also configure this command to run automatically. By default, Flex Gateways deployed on Kubernetes have a liveness probe configured. The liveness probe periodically runs the liveness check command and automatically restarts the Flex Gateway pod after a specified number of failed liveness checks. 
+
+For Docker or Linux deployments, there is not a default liveness probe configured to check health or restart unhealthy Flex Gateways. However, you can still use the the liveness check command manually or configure it for automated use with third-party services. Besides the Kubernetes liveness probe, Mulesoft does not provide support for third-party liveness probes or health checks.
+
+An example of how you can use the liveness check command with Docker is by configuring `HEALTHCHECK` in your `docker run` command. For more information about `HEALTHCHECK`, see https://docs.docker.com/engine/reference/run/#healthcheck[Docker run HEALTHCHECK].
+
+//tag::intro2[]
+
+//tag::k8s-liveness-probe[]
+== Before You Begin
+
+Register and deploy Flex Gateway to a Kubernetes cluster. For guidance, see xref:flex-gateway-set-up.adoc[] or xref:flex-gateway-k8-getting-started.adoc[].
+
+== Configure a Kubernetes Liveness Probe
+By default, Flex Gateways running on Kubernetes have a liveness probe configured in the Helm chart. The liveness probe automatically runs the liveness check command at a specified interval and restarts unhealthy pods after a specified number of unhealthy tests. 
```

Comment:
```suggestion
By default, Flex Gateways running on Kubernetes include a liveness probe configured in the Helm chart. The liveness probe automatically runs `flexctl probe --check=liveness` at a specified interval and restarts unhealthy pods after a specified number of failed tests. 
```
Probably good to specify the exact command so that they don't have to scroll back up the page to see it.